### DISCUSSION TITLE
Performance best practices

### DISCUSSION THREAD

**Discussion by ironnysh** [WordPress/developer-blog-content#253] [Topic Ideas]
**Topic:** Website performance
**Audience:** extenders
**Topics covered:** 
- Tips
- Best practices
- How to run live tests using the Performance Team's custom [Google Colab](https://colab.research.google.com/drive/1TmbJhSBuM-gTHA_E_W96pSjraQkFYpyi)

Basically, nudging people to think about performance when they create themes and plugins.

**Background:** 
> Members of the WordPress Performance Team have been working on an introduction to gathering WordPress performance data in the field. This is provided in [this Colab](https://colab.research.google.com/drive/1TmbJhSBuM-gTHA_E_W96pSjraQkFYpyi), as the Colab format is a great fit for this kind of content, since it allows providing BigQuery queries that can be run directly inside of it, and the results can be presented alongside the queries. Colab covers both potential core use-cases as well as plugin or theme use-cases.

See this Make post ([Conducting WordPress performance research in the field](https://make.wordpress.org/core/2024/04/05/conducting-wordpress-performance-research-in-the-field/)) and [this Slack discussion](https://wordpress.slack.com/archives/C02RW657Q/p1712192512716299) on the Training Team channel for more information.

**Comment by ironnysh** [comment]
**Update:** I’ve been chatting about this with @felixarntz, who suggested breaking it into a series:
> Regarding the topics, I think it would be good to cover individual performance aspects in separate articles. What you're describing would be very broad for one article. What might work well is a series, where one intro article paints a holistic overview, and then the other articles go into depth.

**Comment by bph** [comment]
Topic was approved 5/2/24 and conversation continues on the issue
- #258 258
